doc/Extensions/Services.md

```diff
@@ -89,3 +89,10 @@ then you can run the following command to help troubleshoot services.
 ```
 ./check-services.php -d
 ```
+## Service checks polling logic
+
+By default, service checks will be skipped if the associated device is not pingable, and an appropriate entry will be populated in the event log. An exception to this rule are service checks with `IP address` parameter different from the associated device's IP address.
```

Comment:
You can drop the "By default, "  Maybe try to make it a little less wordy.

Reply:
What do you think about below?

```Service check is skipped when the associated device is not pingable, and an appropriate entry is populated in the event log. 
Service check is polled if it's `IP address` parameter is not equal to associated device's IP address, even when the associated device is not pingable.```

Seems simpler like this, I think :)